MapboxNavigation/NavigationViewController.swift

```diff
@@ -793,3 +800,13 @@ extension NavigationViewController: CarPlayConnectionObserver {
         }
     }
 }
+
+// MARK: - MGLMapViewDelegate
+
+extension NavigationViewController: MGLMapViewDelegate {
```

Comment:
I'm not sure that such approach will work in all cases. Since end user can set `MGLMapViewDelegate` delegate it'll override one which was set internally in `NavigationViewController` and current method will never be called. I did not hear any explicit requirement that we should always show buildings extrusion (even without actual navigation, just when showing `NavigationMapView`), so probably calling `setCenter(_:zoomLevel:direction:animated:completionHandler:)` on mapView will be enough to highlight building correctly after arriving to destination. @1ec5, what do you think?

Reply:
RouteMapViewController installs a key-value observer for `MGLMapView.style` instead of registering as an MGLMapViewDelegate. I think NavigationViewController could cleanly do the same.